src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateNotificationService.swift

```diff
@@ -213,6 +245,46 @@ class HealthCertificateNotificationService {
 
 		addNotification(request: request)
 	}
+
+	private func scheduleBoosterNotification(
+		personIdentifier: String,
+		completion: (() -> Void)? = nil
+	) {
+		Log.info("Schedule booster notification for person with id: \(private: personIdentifier)", log: .vaccination)
+
+		let content = UNMutableNotificationContent()
+		content.title = AppStrings.LocalNotifications.certificateGenericTitle
+		content.body = AppStrings.LocalNotifications.certificateGenericBody
+		content.sound = .default
+
+		let request = UNNotificationRequest(
+			identifier: LocalNotificationIdentifier.boosterVaccination.rawValue + "\(personIdentifier)",
+			content: content,
+			trigger: nil
+		)
+
+		addNotification(request: request, completion: completion)
+	}
+
+	private func scheduleCertificateReissuanceNotification(
+		personIdentifier: String,
+		completion: (() -> Void)? = nil
+	) {
+		Log.info("Schedule certificate reissuance notification for person with id: \(private: personIdentifier)", log: .vaccination)
+
+		let content = UNMutableNotificationContent()
+		content.title = AppStrings.LocalNotifications.certificateGenericTitle
```

Comment:
Are the notifications looking the same or is it a mistake?

Reply:
Very good question! The tile was correct, but I got the message wrong, fixed it now :)